Tests/Unit/Transaction/TransactionTests.swift

```diff
@@ -80,7 +83,8 @@ class TransactionTests: XCTestCase {
             fee: fixture.fee,
             tombstoneBlockIndex: fixture.tombstoneBlockIndex,
             fogResolver: fixture.fogResolver,
-            blockVersion: .minRTHEnabled).get()
+            blockVersion: .minRTHEnabled,
+            rng: TestRng()).get()
```

Comment:
nit: this change isn't necessary because it should be able to use a default value right ?